universal-login-example/test/services/BackupService.js

```diff
@@ -0,0 +1,31 @@
+import chai, {expect} from 'chai';
+import Services from '../../src/services/Services';
+import BackupService from '../../src/services/BackupService';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+describe('BackupService', async () => {
+  let backupService;
+  let identityService;
+
+  before(async () => {
+    identityService = [];
+    identityService.identity = [];
+    identityService.identity.name = 'kyle';
+    backupService = new BackupService(identityService);
+  });
+  describe('Backup Codes', async () => {
+    it('generates one backup code', async () => {
+      await backupService.generateBackupCodes(1);
+      expect(backupService.backupCodes.length).to.eq(1);
```

Comment:
An expect on the backup code itself would be good. 
I would propose simple regexp to check content and length.